soc/realtek/ameba/common/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024 Realtek Semiconductor Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(.)
```

Comment:
Huh? You have a common folder with just a cmake file and include this directory, which is empty?